third_party/conan/recipes/crashpad/patches/windows_adaptions.patch

```diff
@@ -23,18 +26,18 @@ index d8dc6cd..4f55b5f 100644
 +      cflags = [ "/MTd" ]
 +    }
 +  }
- }
- 
- config("release") {
-@@ -64,7 +77,6 @@ config("release") {
+   if (!mini_chromium_is_win) {
+     cflags = [ "-g" ]
```

Comment:
can you explain why we need "-g" in a release build? Probably because there is no relwithdebinfo build, right?

Reply:
These lines are part of the original BUILD.gn file (3 lines after the change); its content have changed since version `20191105`. I tried to leave the logic of the patch, and just regenerated the patch with new version of BUILD.gn file adding exactly those lines that were in the patch.

Reply:
Ah sorry. You're right. It's hard to read a diff of a diff. :innocent: